docs/DesignDocs/libc_initialization.md

```diff
@@ -0,0 +1,169 @@
+# Musl libc Initialization
+
+## Normal musl libc initialization on Linux x86\_64
```

Comment:
What about on Windows?  Should there be another section?  Or can this section be made OS-agnostic?

Reply:
This is for in-enclave libc initialization and OE uses linux paradigms inside the enclave regardless of the host OS. I can specify this somewhere.